integration-tests/src/test/groovy/uk/gov/justice/digital/hmpps/cattool/DashboardWomenSpecification.groovy

```diff
@@ -0,0 +1,156 @@
+package uk.gov.justice.digital.hmpps.cattool
```

Comment:
There will be more changes in 1244, please keep this branch updated with CAT-1244 changes (merge 1244 into this branch). I assume this will be extended with female recat tests as well.

Reply:
Yes, there will be tests for recat as well.Will update those comments when I start working for recat test